docs/console/project-configuration/project-settings.md

```diff
@@ -68,13 +68,45 @@ An imagePullSecret is a Kubernetes entity used to authenticate and authorize acc
 
 This card shows the user, in the form of a table, the list of names that will be used in deployment and cronjob files for pulling Docker images. The user has the option of adding new names through the "Add image pull secret" button, which will open a simple modal asking for the secret name. Similarly, a secret can be edited and deleted using the appropriate icons at the respective row in the table.
 
-### Runtime settings
+### Deployment options
+
+This card allows users to manage some options regarding the Project deployment.
+
+![runtime settings](./img/settings-deployment-options.png)
+
+- **Deployment strategy**: specifies whether the Project deployment follows a [GitOps-based approach](/development_suite/deploy/gitops-based/index.md) or a [pipeline-based approach](/development_suite/deploy/pipeline-based/index.md).
+
+  :::info
+  The choice of the deployment strategy is only available on Projects with the [Enhanced Workflow](/development_suite/set-up-infrastructure/enhanced-project-workflow.md) enabled. Standard Projects only support a pipeline-based deployment strategy.
+  :::
+  
+    - In **Pull mode**, through a GitOps tool connection, Project configurations saved on the Git repository are always consistent with the Project’s Runtime configurations
+
+    - In **Push mode**, updates and changes to applications or configurations are initiated and directly pushed from a CI/CD pipeline to the target environment
+
+- **Pipeline runner**: specifies the command line deployment tool used by the Project. This option is not available for Projects that adopt a "Pull" deployment strategy, where no pipeline runner is needed. The chosen tool can either be [Mia-Platform LaunchPad (MLP)](https://github.com/mia-platform/mlp), or another custom tool. 
+  :::info
+  In order to use `mlp`, it is required to have the [Smart Deploy](/development_suite/deploy/overview.md#smart-deploy) feature enabled.
+  :::
```

Comment:
this is not true, it's actually the other way around; using mlp enables the smart deploy feature

I'd remove the info box entirely

```suggestion
```